pkg/jaeger/README.md

```diff
@@ -0,0 +1,36 @@
+# Jaeger
+
+This package contains filter for generating `jaeger span`
+
+## Usage
+
+### Importing
+
+```go
+import "github.com/AccelByte/go-restful-plugins/v3/pkg/jaeger"
+```
+
+### Filter
+
+Filter is restful.FilterFunction for generating jaeger span using zipkin headers
+
+#### Example usage of filter for all endpoints
+
+```go
+    span := GetSpanFromRestfulContext(request.Request.Context())
```

Comment:
is the span returned here is the span injected in the filter?
or will it be a new child span?
because if it is the same as the span injected in the filter we don't need to have the defer span.Finish() below since the span already finished in Finish(span) called in the filter

Reply:
right, the `defer` already defined in the Filter as we just transport the existing jaeger-span